server/server.go

```diff
@@ -505,21 +507,32 @@ func (s *Server) configureAccounts() error {
 	return nil
 }
 
-// Setup the memory resolver, make sure the JWTs are properly formed but do not
-// enforce expiration etc.
+// Setup the account resolver. For memory resolver, make sure the JWTs are
+// properly formed but do not enforce expiration etc.
```

Comment:
Why would we not enforce expiration?

Reply:
Not sure. This is invoked in startup and during config reload, if that brings memory back. You added this comment in that commit: https://github.com/nats-io/nats-server/commit/5b42b99dc1713907699c46310eede6359577b348

Reply:
Yes but you added code that verifies them now IIRC.

Reply:
No, the only change is to add the TLSConfig if specified (GH diff makes it look really different, but that is misleading). Basically only this is added:
```
// For URL resolver, set the TLSConfig if specified.
if opts.AccountResolverTLSConfig != nil {
    if ar, ok := opts.AccountResolver.(*URLAccResolver); ok {
        if t, ok := ar.c.Transport.(*http.Transport); ok {
	     t.CloseIdleConnections()
             t.TLSClientConfig = opts.AccountResolverTLSConfig.Clone()
        }
     }
}
```

Reply:
hmm ok. I should look at this at some point, we want to honor things like expiration etc.